src/Cake.Core.Tests/Unit/IO/DirectoryPathTests.cs

```diff
@@ -255,6 +255,25 @@ public void Can_Not_Combine_Directory_Path_With_Absolute_Windows_Directory_Path(
             }
         }
 
+        public sealed class TheGetParentMethod
+        {
+            [WindowsTheory]
```

Comment:
This looks good. We'll need a set of tests that run cross-platform too, covering non-Windows paths as well